src/NHSD.BuyingCatalogue.Ordering.Api/Models/GetOrderItemModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public sealed class GetOrderItemModel
+    {
+        public ServiceRecipientModel ServiceRecipient { get; set; }
+
+        public string CatalogueSolutionId { get; set; }
+
+        public string CatalogueSolutionName { get; set; }
+
+        public DateTime? DeliveryDate { get; set; }
+
+        public int Quantity { get; set; }
+
+        public string EstimationPeriod { get; set; }
+
+        public string ProvisioningType { get; set; }
+
+        public string Type { get; set; }
+
+        public string CurrencyCode { get; set; }
+
+        public ItemUnitModel ItemUnitModel { get; set; }
```

Comment:
This should be ItemUnit not ItemUnitModel, for the property name

Reply:
👍